src/host/main.cpp

```diff
@@ -318,6 +318,14 @@ int main(int argc, char** argv)
     "Subject Alternative Name in node certificate. Can be either "
     "iPAddress:xxx.xxx.xxx.xxx, or dNSName:sub.domain.tld");
 
+  size_t jwt_key_refresh_interval_s = 1800;
```

Comment:
What motivated the default for this value? I can't seem to find how often JWKS document are typically updated and how long the overlap between the old and new keys is.

Reply:
It's generally one day or more, so this is very conservative. 1800 comes from https://github.com/AzureAD/passport-azure-ad/blob/5ce0f199af305ab3568facdfa4a4d31156d5bc2b/lib/bearerstrategy.js#L42